Core/NoOpRunner.Core/GamePlatforms.cs

```diff
@@ -61,6 +62,7 @@ public void ClickShape(int x, int y)
                     }
                     else
                     {
+                        //so? now it's just platforms, we don't care
```

Comment:
This exception is useful because currently there should be no shape overlaps. It'll inform us if we i.e. fuck up platform generation

Reply:
Platforms collide sometimes,

Reply:
Yeah, but they should not, so we should fix the platform generation, not remove the exception